extensions/interactions/NumericInput/directives/numeric-input-validation.service.ts

```diff
@@ -160,35 +160,40 @@ export class NumericInputValidationService {
     return warningsList;
   }
 
-  getErrorString(value: string): string {
-    if (!value) {
-      return '';
+  getErrorString(value: number): string {
+    if (value === undefined || value === null) {
```

Comment:
Since the input type has been changed to "number", the validation will automatically fail when the input value is not a valid number -- it causes a form error and sets the localValue to undefined/null (see https://docs.angularjs.org/error/ngModel/numfmt, https://docs.angularjs.org/api/ng/input/input%5Bnumber%5D). There does not seem to be a way to obtain the invalid number entered in order to parse and provide a more specific message. This is because the sanitization of invalid numbers is done internally (https://html.spec.whatwg.org/multipage/input.html#number-state-(type=number)).
> The value sanitization algorithm is as follows: If the value of the element is not a valid floating-point number, then set it to the empty string instead.